variables.tf

```diff
@@ -310,6 +310,16 @@ variable "dns_forwarders" {
     default   = "8.8.8.8; 8.8.4.4"
 }
 
+variable "lb_ipaddr" {
+    description = "Define the preconfigured external Load Balancer"
+    default = ""    
+}
+
+variable "ext_dns" {
+    description = "Define the preconfigured external DNS and Load Balancer"
+    default = ""
+}
+
```

Comment:
update to the helpernode_tag ?

Reply:
There is no tag on ocp4-helpernode, the PR will be merged in master.

Reply:
Was referring to this - https://github.com/ocp-power-automation/ocp4-upi-powervm/blob/cab69442598ed00c63ba141cb6ba107f6600f927/variables.tf#L243 

We should wait for the helpernode PR to be available in main/master branch before committing this.

Reply:
@cs-zhang the changes need to be merged in helpernode first before we can use it here.

Reply:
Christian said the PR will be merged in next week, we can do after it.